libwebthree/WebThree.cpp

```diff
@@ -28,13 +28,10 @@
 #include <libethereum/Defaults.h>
 #include <libethereum/EthereumHost.h>
 #include <libwhisper/WhisperHost.h>
+#include <libethereum/ClientTest.h>
 #include <libethashseal/EthashClient.h>
-#if ETH_AFTER_REPOSITORY_MERGE
```

Comment:
It looks like this edit breaks these #ifdefs, which we need for bridging the repo-reorg (where the granularity of the buiildinfo.h files changes.   Please could they be restored?   Thanks!